airbyte-integrations/connectors/source-tweag-blog/integration_tests/invalid_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "todo-wrong-field": "this should be an incomplete config file, used in standard tests"
```

Comment:
Should it be changed?

Reply:
Removed in https://github.com/tweag/airbyte/pull/2/commits/ad3cd0bf4105637948152fc7dccdf39c5ca5e6b5